### ISSUE TITLE
Bouckaert et al. 2018

### ISSUE THREAD

**Issue by SimonGreenhill** [lexibank/lexibank#195]
Updated version of bowernpny - https://zenodo.org/record/1320471

**Comment by LinguList** [comment]
This should probably not be "bowernpny", but rather "bouckaertpny", since the versions are quite different, and belong to two different papers.

**Comment by SimonGreenhill** [comment]
Yes, sorry -- I meant that this should be a new dataset that derives bowernpny

**Comment by LinguList** [comment]
Yes, we can recycle quite a lot of the code, I guess (or the profile, etc.)
